src/Mod/Part/Gui/TaskAttacher.cpp

```diff
@@ -65,12 +65,12 @@ const QString makeRefString(const App::DocumentObject* obj, const std::string& s
     if (!obj)
         return QObject::tr("No reference selected");
 
-    if (obj->isDerivedFrom<App::OriginFeature>() ||
+    else if (obj->isDerivedFrom<App::OriginFeature>() ||

```

Comment:
What linter is giving you an error here? This looks like it is adding a linter issue rather than removing one.